### ISSUE TITLE
All files & folders collapsed in Explorer view on start-up

### ISSUE THREAD

**Issue by glen-84** [microsoft/vscode#153245]
Issue Type: <b>Bug</b>

Whenever I restart VS Code, all files and folders are collapsed in the Explorer view.

- I think this started happening in the latest release.
- I seem to remember something like this happening in the past.

VS Code version: Code 1.68.1 (30d9c6cd9483b2cc586687151bcbcd635f373630, 2022-06-14T12:48:58.283Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No
Remote OS version: Linux x64 5.10.16.3-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-9400F CPU @ 2.90GHz (6 x 2904)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.92GB (17.87GB free)|
|Process Argv|--folder-uri vscode-remote://wsl%2Bdebian/home/glen/development-environment/projects/ApiNext --crash-reporter-id 0da054fe-3726-474a-bc0d-7fe4a5e5fb13|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Debian|
|OS|Linux x64 5.10.16.3-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i5-9400F CPU @ 2.90GHz (6 x 2904)|
|Memory (System)|15.59GB (12.09GB free)|
|VM|0%|
</details><details><summary>Extensions (31)</summary>

Extension|Author (truncated)|Version
---|---|---
auto-close-tag|for|0.5.14
eml|lei|0.4.0
dotenv|mik|1.0.1
remote-wsl|ms-|0.66.3
JavaScriptSnippets|xab|1.8.0
toggle-hidden|adr|1.0.2
vscode-intelephense-client|bme|1.8.2
vscode-svgviewer|css|2.0.0
vscode-eslint|dba|2.2.2
EditorConfig|Edi|0.16.4
vscode-npm-script|eg2|0.3.25
prettier-vscode|esb|9.5.0
vscode-highlight|fab|1.6.0
auto-close-tag|for|0.5.14
psalm-vscode-plugin|get|2.6.0
todo-tree|Gru|0.0.215
eml|lei|0.4.0
dotenv|mik|1.0.1
csharp|ms-|1.25.0
vscode-jest|Ort|4.6.0
vscode-commons|red|0.0.6
vscode-xml|red|0.20.0
vscode-yaml|red|1.8.0
vscode-sort-json|ric|1.20.0
trailing-spaces|sha|0.3.1
code-spell-checker|str|2.2.5
vscode-stylelint|sty|1.2.2
vscode-phpsab|val|0.0.15
gitblame|wad|8.2.3
change-case|wma|1.0.0
JavaScriptSnippets|xab|1.8.0

(2 theme extensions excluded)

</details>

**Comment by glen-84** [comment]
No, I think it only happens with WSL folders, but it doesn't seem to affect a _workspace_ in WSL.

I've tried with `--disable-extensions` and that doesn't seem to help.

**Comment by lramos15** [comment]
> No, I think it only happens with WSL folders, but it doesn't seem to affect a _workspace_ in WSL.

So it only happens when you open a folder that is mounted in WSL? You aren't connected to the WSL instance via the remote WSL extension?

**Comment by lramos15** [comment]
> Is there some kind of UI cache that can be reset?

you can try `code --user-data-dir some/folder  --extensions-dir some/folder ` just to see if a fresh install of vs code with none of your user state could reproduce this

**Comment by glen-84** [comment]
> $ code --user-data-dir vsctmp --extensions-dir vsctmp projects/ApiNext/
Ignoring option 'user-data-dir': not supported for code.
Ignoring option 'extensions-dir': not supported for code.

**Comment by lramos15** [comment]
Was able to reproduce this. Will investigate

**Comment by glen-84** [comment]
@lramos15 Any update on this? It's really quite annoying.
